### ISSUE TITLE
Luma PLY files not working?

### ISSUE THREAD

**Issue by alexmgee** [playcanvas/super-splat#12]
Having some difficulty getting any of the downloadable PLY files from LumaLabs to work with your editor. When adding the file, it seems to indicate that processing is happening but then the screen remains empty. 

Have attempted a few different scenes using each of the various PLY download options available.

**Comment by slimbuck** [comment]
Hi,  does the browser console shows any errors?

**Comment by liorazi** [comment]
@alexmgee 

I think you may have used the wrong option in the Downloads page:
![Screenshot 2023-11-04 at 12 21 48](https://github.com/playcanvas/super-splat/assets/2930153/a7138476-8c73-46f4-a27a-6767a308d9ee)

Be sure you choose the "Gaussian Splat" PLY and not the Scene PLY one as shown in the screenshot

**Comment by alexmgee** [comment]
@liorazi I understand the difference. As I said in the post, I used all available forms of PLY files from Luma. 

@slimbuck I see no errors. When dropping or adding the file, its name will appear in the lower right next to the 'Choose File' button... but nothing ever appears. 

Since posting this issue last night I have tested multiple other captures and did finally have some success. I'm not sure what the reason for some working and others  not working is, but I am guessing it's on Luma's end. Downloading the GS PLY files results in a zipped download containing the .ply file. Some of the downloads result in a file named 'gaussian_splatting_point_cloud.py' inside of a folder with the appropriate capture name. These do not seem to work. Other captures will return a zipped file with only a .ply file inside, with the  naming convention 'gs_CAPTURE_NAME_HERE.ply'. These files seem to work fine. 

Thanks for the help and for this awesome tool.
